guardian/fsbp-tools
# Update project READMEs

## What does this change?

- bucket-blocker
  - Make stylisation of `bucket-blocker` consistent
  - Add a note about how to generate hashes
- ingress-inquisition
  - Add installation instructions for brew
  - Add release instructions for brew
  - Add a note about how to generate hashes

## How to test

Check the links. Make sure the download/install steps work. NB this will not work until several minutes after [this PR](https://github.com/guardian/homebrew-devtools/pull/104) has been merged
